css/lavish-bootstrap.css

```diff
@@ -1,6841 +0,0 @@
-/* CSS generated by http://lavishbootstrap.com */
```

Comment:
Same question, you removed this file but I remember that some of those codes have an effect on the website, so where have you inserted the code?

Reply:
lavish-bootstrap.css doesn’t seem to be used—only lavish-bootstrap.min.css (which, ironically, isn’t even minified). To clean up, this update removes the unused lavish-bootstrap.css, and adds a TODO to later rename and minify lavish-bootstrap.min.css.

Reply:
I will also check this PR to make sure and approve after testing to make sure